### DISCUSSION TITLE
Projects with low Build Activity - Survey

### DISCUSSION THREAD

**Discussion by jadsonjs** [gogs/gogs#6693] [General]
Dear Developers of project **gogs/gogs**,

We are conducting a study on CI metrics and we noticed that in your project, something peculiar happened.

During the period you are using Travis CI, the project passed by a long period of inactivity.
We noticed that the TravisCI configuration file (.travis.yml) was committed to your project on the date of **2014-05-07**.
But there is only a build history in TravisCI for a very short period, between **2020-03-10** and **2020-03-27**, in which were performed suddenly **100** builds.

Please, we would like you to answer this short survey with just 4 questions to help us identify the cause of this behavior.

https://forms.gle/UkkFrxpi7MNscYmV7

**Comment by unknwon** [comment]
For anyone else looking at this thread, I want to make it clear that from my POV it was Travis CI abandoned us and we just stopped using it :) Thus migrated to GitHub Actions.
